### ISSUE TITLE
While importing tcv it crashes

### ISSUE THREAD

**Issue by KnedlOF** [hay-kot/homebox#588]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### Homebox Version

v0.10.1

### What is the issue you are experiencing?

When I import my tcv with all items (over 150), it seems like it crashes and returns me to login page. When I log in again, there will only be one location and tag added, thats it, nothing else. I exported my tcv from my previous settup and I am not sure what version it was. I tried on my computer with version 0.9.2 and it worked perfectly fine. 

### How can the maintainer reproduce the issue?

I can't import my items. 

### Deployment

Docker (Linux)

**Comment by KnedlOF** [comment]
I tried on my new setup (Linux Debian) diffirent versions, but it still had the same problem. I managed to import at the end. The issue was when you press submit first time nothing happens, but when you press it again it fails. So I figured out if I wait some time after pressing first time it will import. A little bit sketchy but thats how I got it to work.

**Comment by hay-kot** [comment]
Hey! I'd like to look into this this weekend, would you be willing to share your tsv file with me so I can attempt to reproduce the error? Also, if you have any log files that would be helpful.

**Comment by KnedlOF** [comment]
[homebox-items.zip](https://github.com/hay-kot/homebox/files/13374055/homebox-items.zip)

**Comment by hay-kot** [comment]
This works fine for me on the latest version. Could you try upgrading to the most recent release? If you still see the issue, and logs would be helpful in debugging the problem.

**Comment by MGM03** [comment]
Hey sorry to join but I am having a very similar issue with both TSV and CSV, it gets like 3 items in sometimes 10 if im lucky before this shows up in logs (see screenshot) and I get kicked to the login page where I login and see a few of the items show up
![Screenshot 2023-11-18 104359](https://github.com/hay-kot/homebox/assets/108984525/fe3fd6ae-d3fc-4af1-8c00-f83691ca1d05)
I will attach a copy of the CSV im using but every TSV and CSV i use has the same issue including an export freshly made by homebox
[PS4.csv](https://github.com/hay-kot/homebox/files/13400144/PS4.csv)

Side Note: The Import References seem to be ignored aswell, It makes a duplicate

For context I use TrueNAS Scale (Debian Linux) and the issue I face is both with the truecharts version as well as pulling the docker container directly

Note added just before commenting:
For some reason when i create new users sometimes it will allow me to fully import KnedlOF's TSV but not my attached CSV, The usernames have to be never used before (even if you deleted the previous profile) and only then will you get a chance at it deciding to work

**Comment by ylixium** [comment]
I have experienced the same issue as well only the first item is being imported. It seems `HB.import_ref`, `GB.location`, `HB.labes` are ignored. Changes made to the first item asset, name quality and description did imported to the database. I am using v0.10.2.
I found the following error on my logs:
![chrome_1700430339](https://github.com/hay-kot/homebox/assets/99920424/c3f977fc-231e-450e-87bb-df72a4e6d1f4)
